examples/next/pages/ui/components/liveness/with-credential-provider/index.page.tsx

```diff
@@ -0,0 +1,34 @@
+import dynamic from 'next/dynamic';
+import React from 'react';
+
+import { Amplify } from 'aws-amplify';
+import { fromCognitoIdentityPool } from '@aws-sdk/credential-providers';
+import awsExports from '@environments/liveness/liveness-environment/src/aws-exports';
```

Comment:
Instead of directly importing from `@environments`, can you create a local `aws-exports.js` like we did here? https://github.com/aws-amplify/amplify-ui/blob/6768e978c64fa80bfcc6b5aeb38aa944fa50eb3f/examples/next/pages/ui/components/authenticator/reset-password/aws-exports.js#L1-L2